File: Week6/sentimental-mario-more/mario.py
from cs50 import get_int

height = get_int("Height: ")
if (height < 1 or height > 8):
    height = get_int("Height: ")
for i in range(height):
    for a in range(height-i-1):
        print(" ", end="")
    for a in range(i+1):
        print("#", end="")
    print("  ", end="")
    for a in range(i+1):
        print("#", end="")
    print("")

File: Week1/mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void) {
    int height = get_int("Height: ");
    while (height <= 0 || height > 8) {
        height = get_int("Height: ");
    }

    for (int i = 0; i < height; i++) {
        for(int a = 0; a < height-i-1; a++) {
            printf(" ");
        }
        for (int a = 0; a <= i; a++) {
            printf("#");
        }
        printf("  ");
        for(int a = 0; a <= i; a++) {
            printf("#");
        }
        printf("\n");
        //printf("%*c%  -*c\n", height, '#', height, '#');
    }
}